### PR TITLE
Add demo video

### PR DESCRIPTION
**Pull request by scratchscratchscratchy**
not sure it's optimal to add the gigantic video twice - but the UX is better than using youtube I feel... 

I tested page scaling of video in Chrome & Firefox dektop. 

image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -167,6 +167,20 @@ <h4 class="feature-title mt-24">Receive back BTC</h4>
...
 
+            <section class="pricing section">
+                <div class="pricing-header text-center">
+                    <h2 class="section-title mt-0">Demo version live on Bitcoin testnet</h2>
+                    <p class="section-paragraph mb-0"></p>
+                    <div class="hero-cta"><a class="button button-primary" href="https://itchysats.medium.com/p2p-bitcoin-cfds-give-it-a-try-4db2d5804328" target="_blank">Try it yourself!</a></div>
+                </div>
+                <div class="container">
+                    <video width="100%" controls autoplay>
+                        <source src="dist/images/screen_record_demo.mov" type="video/mp4">
```
Comment (1010Tom) [inline index.html]: Can we use a youtube link instead? 

The problem with an embedded video is that it will slows down the website

Comment (scratchscratchscratchy) [reply index.html]: Can do, but I find the youtube embedded iframe ugly end distracting. I think with a modern browser using HTML5 videos should not be an issue, but maybe I underestimate optimising for different devices n' stuff :D 

We will need some CSS magic to scale the iframe properly I found some and slapped it in...

Comment (scratchscratchscratchy) [reply index.html]: image

Comment (1010Tom) [reply index.html]: Thanks, I think that looks good :) well done.

**Review by 1010Tom — APPROVED** [review]
👍

**Pull request merged**